runtime/scripts.vim

```diff
@@ -245,7 +245,7 @@ else
     set ft=xhtml
 
     " HTML (e.g.: <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN")
-  elseif s:line1 =~? '\<DOCTYPE\s\+html\>'
+  elseif s:line1 =~? '\<<!DOCTYPE\s\+html\>'
```

Comment:
I think `<!DOCTYPE\s+html\>` is enough. Adding `\<` is redundant in this case.
If you want to avoid some extra characters before the DOCTYPE, maybe `^\s*<!DOCTYPE\s\+html\>` is better. (I'm not sure the extra characters before the DOCTYPE are accepted or not.)

BTW, the behavior of `\<` actually depends on the option `'iskeyword'`. If `'iskeyword'` includes `<`, `<` is considered as a word character.